app/views/pages/confirm-profile.ejs

```diff
@@ -1,143 +1,103 @@
-                <div class="row">
-                  <div class="col-12">
-                    <input type="hidden" name="success" value="true" />
-                    <% if (!hasCookie) { %>
-                    <input required type="checkbox" id="accept" name="accept" />
-                    <label for="accept" class="normal-font"
-                      ><%= __("I've noted down the link.") %></label
-                    >
-                    <% } %>
-                  </div>
-                  <div class="col-12">
-                    <ul class="actions">
-                      <li>
-                        <input
-                          class="inverse"
-                          type="submit"
-                          value="<%= __('Proceed') %>"
-                        />
-                      </li>
-                    </ul>
-                  </div>
-                </div>
-              </form>
-            </div>
-          </div>
-        </div>
-      </section>
-    </div>
-    <%- include('partials/footer') -%>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.4/clipboard.min.js"></script>
-    <script>
-      // Remove query params from url
-      if (window.history.replaceState) {
-        window.history.replaceState(
-          {},
-          document.title,
-          window.location.pathname
-        );
-      }
-
-      var clipboard = new ClipboardJS('#profile-url-id');
-      var statusText = document.getElementById('profile-copy-status-text');
-
-      function setStatusAlert(element, text, alert_class) {
-        txt = document.createTextNode(text);
-        element.appendChild(txt);
-        element.style.display = 'block';
-        element.classList.add(alert_class);
-      }
-
-      clipboard.on('success', function(e) {
-        if (statusText.style.display == 'none') {
-          setStatusAlert(
-            statusText,
-            '<%= __("The link has been copied.") %>',
-            'alert-success'
-          );
-        }
-      });
-
-      clipboard.on('error', function(e) {
-        if (statusText.style.display == 'none') {
-          setStatusAlert(
-            statusText,
-            '<%= __("Could not copy the link, please copy it yourself") %>',
-            'alert-error'
-          );
-        }
-      });
-    </script>
-
-    <script>
-      // This is done so that we can show a hint on the front page that a profile might exist
-      // when entering from the same browser at a later time.
-      var hasCookie = 'true' === '<%= hasCookie %>';
-      if (!hasCookie) {
-        window.localStorage.setItem('isUsingPasscodeLink', 'true');
-      }
-    </script>
-  </body>
-</html>
+<% const { hasCookie = false, passcode = 'unkown' } = locals; %>
+<!-- -->
+<%- include('partials/header') -%>
+
+<section
+  class="max-w-xl sm:mt-20 mx-auto sm:bg-cvs-green sm:rounded-lg py-4 sm:p-10 mx-3"
+>
+  <% if (hasCookie) { %>
```

Comment:
We can consider skipping the confirmation page altogether and going straight to the profile if the user says yes to the cookie. But we can look at that later, no need to fix it in this PR.

Reply:
It kind of nice to have them save the code when they use their computer with multiple users, right? We should add multiple users to the browser storage I think so people can do this:

![CleanShot 2020-03-24 at 10 32 14@2x](https://user-images.githubusercontent.com/1079135/77410144-c1bdaf80-6dba-11ea-905f-2c32b7fa40f8.jpg)